### DISCUSSION TITLE
Internal compiler errors with arm-none-eabi-gcc 11.2-2022.02 on Macs with Apple Silicon

### DISCUSSION THREAD

**Discussion by SloBloLabs** [Homebrew/discussions#3093] [Casks]
Hi Community,
I would like to bring an issue to your attention regarding arm-none-eabi-gcc version 11.2 cask. It turns out to cause issues on M1 Macs and is also reflected here: https://community.arm.com/support-forums/f/compilers-and-libraries-forum/52377/internal-compiler-errors-with-arm-none-eabi-gcc-11-2-2022-02-on-macs-with-apple-silicon
Using version 10.3 the issue did not show up. So my request would be to enable version choosing on the cask so you can easily switch between the versions until a fix becomes available. Today only a manual step back is possible (as far as I can see). Is there a better procedure to hand in such requests for homebrew aside of this discussion area? An official homebrew issue does not appear adequate to me.
Thanks for any suggestions or hints.
Oliver

**Comment by SMillerDev** [comment]
The way for feature requests in Homebrew is to make a pull request

**Reply by SloBloLabs** [reply]
Well, a pull request comes with a piece of code to solve it, which I don't have. What I want is to report an issue together with a remediation proposition.
